client/src/components/trip-list-container/TripListContainer.jsx

```diff
@@ -32,6 +36,13 @@ export default function TripListContainer({
     setDisplayBtn(true);
   };
 
+  const handleLogout = () => {
```

Comment:
Is it enough to back to login page?

Reply:
Do you want to redirect somewhere else? It's working going back to /